src/main/java/pl/futurecollars/invoicing/utils/FilesService.java

```diff
@@ -0,0 +1,50 @@
+package pl.futurecollars.invoicing.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
+public class FilesService {
+
+  public void createRepository(Path idPath, Path databasePath) throws IOException {
```

Comment:
Tę metodę również bym zmienił na coś bardziej ogólnego. Ta klasa nie może znać kontekstu logiki biznesowej. To tylko narzędzie do obsługi plików niezależne od kontekstu.

Więc tutaj jak i w metodach poniżej należy zmienić nazewnictwo.